search/search_repository.go

```diff
@@ -777,7 +806,7 @@ func (r *GormSearchRepository) Filter(ctx context.Context, rawFilterString strin
 	// parse
 	// generateSearchQuery
 	// ....
-	exp, err := parseFilterString(ctx, rawFilterString)
+	exp, _, err := parseFilterString(ctx, rawFilterString)
```

Comment:
@baijum could you please explain again, if you are not using query options (q.Options), then why there is a need to pass it in return statement in `parseFilterString`?

Reply:
I have added it for future. Will remove it.

Reply:
Removed. 48e74f3d5aa3b0668fc07067e33be8b65e538b73